encoders/nlp/VSETextEncoder/manifest.yml

```diff
@@ -8,7 +8,7 @@ description: |
 url: https://jina.ai
 vendor: Jina AI Limited
 documentation: https://github.com/jina-ai/jina-hub
-version: 0.0.1
+version: 0.0.2
```

Comment:
the previous image version is `0.0.3`

Reply:
Yes it was just a test I will change in there.